### PR TITLE
more changes for events and timelines

### PR DESCRIPTION
**Pull request by marcel-blonk**
Made events more explicit. Fixed timelines.

### CODE REVIEW

File: lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/query/jdo/SqlConstants.java [hunk 1]
```diff
@@ -601,10 +601,11 @@
 	public static final String COL_PARTICIPANT_DATA_DESCRIPTOR_ID = "ID";
 	public static final String COL_PARTICIPANT_DATA_DESCRIPTOR_NAME = "NAME";
 	public static final String COL_PARTICIPANT_DATA_DESCRIPTOR_DESCRIPTION = "DESCRIPTION";
+	public static final String COL_PARTICIPANT_DATA_DESCRIPTOR_TYPE = "TYPE";
 	public static final String COL_PARTICIPANT_DATA_DESCRIPTOR_REPEAT_TYPE = "REPEAT_TYPE";
 	public static final String COL_PARTICIPANT_DATA_DESCRIPTOR_FREQUENCY = "FREQUENCY";
 	public static final String COL_PARTICIPANT_DATA_DESCRIPTOR_DATETIME_START_COLUMN_NAME = "DATETIME_START_COLUMN_NAME";
-	public static final String COL_PARTICIPANT_DATA_DESCRIPTOR_DATETIME_END_COLUMN_NAME = "DATETIME_END_COLUMN_NAME";
+	public static final String COL_PARTICIPANT_DATA_DESCRIPTOR_EVENT_COLUMN_NAME = "EVENT_COLUMN_NAME";
```
Comment (alxdarksage) [inline lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/query/jdo/SqlConstants.java:608]: How will this work? A time range is start to event date, an event is just the event date? Or their both the same date if the event is a moment in time?

File: lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/bridge/model/data/ParticipantDataDescriptor.json [hunk 2]
```diff
@@ -40,9 +44,9 @@
 			"type": "string",
 			"description": "The (optional) name of column that holds the point of time or start of a time range used in time series"
 		},
-        "datetimeEndColumnName": {
+        "eventColumnName": {
 			"type": "string",
-			"description": "The (optional) name of column that holds the end of a time range used in time series"
```
Comment (alxdarksage) [inline lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/bridge/model/data/ParticipantDataDescriptor.json:45]: looks like event == end time

File: lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/query/jdo/SqlConstants.java (hunk 1 above)
Comment (marcel-blonk) [inline lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/query/jdo/SqlConstants.java:608]: end == start if it is a moment in time

File: lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/bridge/model/data/ParticipantDataDescriptor.json (hunk 2 above)
Comment (marcel-blonk) [inline lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/bridge/model/data/ParticipantDataDescriptor.json:45]: Event contains name, start and end. datetimeStartColumnName is now only used for point in time for the row (eg. collected_on for CBCs) (probably needs renaming)

**Pull request merged**